README.md

```diff
@@ -42,7 +42,7 @@ python generate_custom_image.py \
 
 *   **--image-name**: The name for custom image.
 *   **--dataproc-version**: The Dataproc version for this custom image to build
-    on. Examples: `1.5.9-debian10`, `1.5.0-RC10-debian10`, `1.5.9-ubuntu18`. For a
+    on. Examples: `1.5.9-debian10`, `1.5.0-RC10-debian10`, `1.5.9-ubuntu18`. Also supports wildcards for minor and patch versions for a specified image family. Examples: `1.5.*-debian10`, `1.*.*-ubuntu18`. For a
```

Comment:
A more consistent version string w/ Dataproc API for this feature will be `1.5-debian10` instead of `1.5.*-debian10`. Also, I do not think that we should support wildcard for minor versions because they are not backward compatible.

Reply:
I've updated the PR to 1) support the `1.5-debian10` format, and 2) removed support for unspecified minor versions. Thanks!